.devcontainer/scripts/dev.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+# Function to display help information.
+show_help() {
+  echo "Statick Developer Commands:"
+  echo "  dev format     - Format all Python code with black and docformatter."
+  echo "  dev help       - Show this help message."
+  echo "  dev qa         - Run linting and static analysis tools."
+  echo "  dev reset-venv - Remove and recreate the virtual environment."
+  echo "  dev test       - Run unit tests and get code coverage."
+}
+
+# Match the `workspaceFolder` in the .devcontainer.json.
+APP_HOME="/opt/statick"
+export APP_HOME
+
+# Function to execute commands
+execute_command() {
+    case $1 in
+    format)
+        cd $APP_HOME || exit
+        black src/statick_tool/ tests/
+        find src/statick_tool/ -name \*.py -exec docformatter -i --black {} \;
+        ;;
+    help)
+        show_help
+        ;;
+    qa)
+        cd $APP_HOME || exit
+        statick . -o /tmp/x -u statick_config/ --profile self_check.yaml --log info
```

Comment:
I prefer the approach of using `--level` instead of `--profile` these days. That results in one less required file anytime you want to add a custom configuration in your own package.

```suggestion
        statick . -o /tmp/x -u statick_config/ --level self_check --log info
```